src/ring_theory/jacobson.lean

```diff
@@ -10,70 +10,87 @@ import ring_theory.localization
 
 /-!
 # Jacobson Rings
-
 The following conditions are equivalent for a ring `R`:
-1. Every radical ideal `I` is equal to its jacobson radical
+1. Every radical ideal `I` is equal to its Jacobson radical
 2. Every radical ideal `I` can be written as an intersection of maximal ideals
-3. Every prime ideal `I` is equal to its jacobson radical
-
+3. Every prime ideal `I` is equal to its Jacobson radical
 Any ring satisfying any of these equivalent conditions is said to be Jacobson.
-
 Some particular examples of Jacobson rings are also proven.
 `is_jacobson_quotient` says that the quotient of a Jacobson ring is Jacobson.
 `is_jacobson_localization` says the localization of a Jacobson ring to a single element is Jacobson.
 `is_jacobson_polynomial_iff_is_jacobson` says polynomials over a Jacobson ring form a Jacobson ring.
-
 ## Main definitions
-
 Let `R` be a commutative ring. Jacobson Rings are defined using the first of the above conditions
-
 * `is_jacobson R` is the proposition that `R` is a Jacobson ring. It is a class,
   implemented as the predicate that for any ideal, `I.radical = I` implies `I.jacobson = I`.
-
```

Comment:
I think it makes the comment more readable if there is an empty line above the header.

Reply:
Added extra line!